### DISCUSSION TITLE
Generate SHA256 hash for releases

### DISCUSSION THREAD

**Discussion by FiYir** [unrealities/MTGA_Draft_17Lands#47] [Ideas]
@unrealities To help ease concerns about the executable, we could include the SHA256 hash for the zip file in the release. Windows has a built-in tool called `certutil` that the build runner can use to generate this hash.

For example, [MTGATracker](https://github.com/mtgatracker/mtgatracker/releases/tag/6.1.3) did something similar with their releases.

What are your thoughts on this?

**Comment by FiYir** [comment]
I added some instructions to [build-windows-exe.yml](https://github.com/FiYir/MTGA_Draft_17Lands/blob/main/.github/workflows/build-windows-exe.yml) that build a hash file and upload it as an artifact. The MTGA_Draft_17Lands_Installer.zip download will include this hash file.

Here's an example.
![image](https://github.com/user-attachments/assets/df438209-2a36-48d9-828e-47aac1ce2c2a)
![image](https://github.com/user-attachments/assets/04a1f737-c007-4ca1-bd4e-e207945730d1)

Would you like me to make a pull request with these changes?

**Reply by unrealities** [reply]
Sounds great. Thank you @FiYir !

**Comment by FiYir** [comment]
@unrealities, could you please add the verification hash section to the 3.27 release description?

Also, the file name is incorrect. It should be MTGA_Draft_Tool_V0327.zip.

![image](https://github.com/user-attachments/assets/059abce9-b86b-4ce8-94cd-e584aa789786)

**Reply by unrealities** [reply]
Thank you for the reminder. I updated the release and changed the file name.

**Comment by FiYir** [comment]
@unrealities, there are some typos in parts of the release description. ```CertUtil -hashfile MTAGA_Draft_Tool_V0327.exe SHA256``` should be ```CertUtil -hashfile MTGA_Draft_Tool_V0327.exe SHA256```.

You can use this description for the 3.27 release:

```
## Verification hash
**Note:** This executable and its SHA256 hash were generated using a [GitHub Action](https://github.com/unrealities/MTGA_Draft_17Lands/actions/workflows/build-windows-exe.yml). To ensure security:

1. **Verify the Hash:** Do not run the executable if the hash does not match. Please contact us immediately if there is any discrepancy.
2. **SHA256 Hash:** `292558685e717037ee3c56fac96c9b34df3a0adcbe6685fe4da0ea1990535bc0`
3. **Verification Instructions:**
   - Unzip the attached file.
   - Open the Windows Command Prompt.
   - Run the following command to compute the SHA256 hash:
     ```bash
     CertUtil -hashfile MTGA_Draft_Tool_V0327.exe SHA256
     ```
   - Compare the resulting hash with the one provided above.
```
